dev-tools/src/lambdas/sqsOperations/handler.ts

```diff
@@ -0,0 +1,15 @@
+import { SqsOperation } from '../../types/sqsOperation'
+import { addMessageToQueue } from './addMessageToQueue'
+
+export const handler = async (params: SqsOperation) => {
+  if (!params) throw Error('Function called with undefined params')
```

Comment:
Could we have a bit more validation than this? As it stands, I could call this function with neither of the right parameters, and it would error out in a confusing way.

Reply:
Done, is that any better?